### ISSUE TITLE
Processes showing 0.0 for CPU% and Mem%

### ISSUE THREAD

**Issue by oskaror** [cjbassi/gotop#124]
Required information:
* gotop -v: 3.0.0
* uname -a: Darwin Kernel Version 17.0.0: Thu Aug 24 21:48:19 PDT 2017; root:xnu 4570.1.46~2/RELEASE_X86_64 x86_64
* Terminal emulator (e.g. iTerm or gnome terminal): iterm2 with Zsh
* Any relevenat hardware info: Macbook Pro 15

![zrzut ekranu 2019-03-07 o 09 48 45](https://user-images.githubusercontent.com/12644040/53943979-ab18bd80-40be-11e9-9d0e-3e61ebf50b16.png)

**Comment by PulsIn** [comment]
Exact same problem as @oskaror on my MacBook Pro, with exactly same configurations (terminal, got version).

It also appears that "Command" listed don't show app activity.
Example : Chrome, Docker, VSCode are never listed in Precesses box.

**Comment by mgamsjager** [comment]
Could be that the shell localization returns eg. `14.4` as `14,4`. The GO `parseFloat` function can't handle that case (expects `.`) and returns a `0` and does not raise an error like you would expect.

**Comment by PulsIn** [comment]
@mgamsjager you are right ! When I try the command `$ ps -ao %cpu`, terminal output `,` float numbers.

So there is no solution to resolve that case, except for changing localization parameters of the OS ?

**Comment by mgamsjager** [comment]
> @mgamsjager you are right ! When I try the command `$ ps -ao %cpu`, terminal output `,` float numbers.
> 
> So there is no solution to resolve that case, except for changing localization parameters of the OS ?

could be that you could initialize the terminal with a certain localisation like said in https://stackoverflow.com/questions/7165108/in-os-x-lion-lang-is-not-set-to-utf-8-how-to-fix-it

if you have `zsh` this command did work for me: `$ zsh --no-rcs --no-global-rcs` to launch a new shell with the default locale

**Comment by cjbassi** [comment]
Just pushed a commit to fix this. Can you test it's working on the master branch?
